app/controllers/api/v1/subscriptions_controller.rb

```diff
@@ -86,11 +91,11 @@ def index
 
       def create_params
         params.require(:subscription)
-          .permit(:external_customer_id, :plan_code, :name, :external_id, :billing_time, :subscription_date)
+          .permit(:external_customer_id, :plan_code, :name, :external_id, :billing_time, :subscription_date, :subscription_at)
```

Comment:
Layout/LineLength: Line is too long. [126/120] (https://rubystyle.guide#max-line-length)